tools/eslint-rules/lowercase-name-for-primitive.js

```diff
@@ -36,8 +36,18 @@ module.exports = function(context) {
     const lowercaseName = name.toLowerCase();
     if (primitives.includes(lowercaseName) && !primitives.includes(name)) {
```

Comment:
`name` could just be replaced with `node.value` now.

Also, pretty sure `!primitives.includes(name)` could be simplified to `name !== lowercaseName`.

Reply:
Made the changes.